tests/test_github_tags.py

```diff
@@ -0,0 +1,29 @@
+# fetchcode is a free software tool from nexB Inc. and others.
+# Visit https://github.com/nexB/fetchcode for support and download.
+#
+# Copyright (c) nexB Inc. and others. All rights reserved.
+# http://nexb.com and http://aboutcode.org
+#
+# This software is licensed under the Apache License version 2.0.
+#
+# You may not use this software except in compliance with the License.
+# You may obtain a copy of the License at: 
+# http://apache.org/licenses/LICENSE-2.0
+# Unless required by applicable law or agreed to in writing, software distributed
+# under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR
+# CONDITIONS OF ANY KIND, either express or implied. See the License for the
+# specific language governing permissions and limitations under the License.
+
+from fetchcode import github_tags
+
+
+def test_github_tags_for_not_empty_list():
+    url = 'https://github.com/nexb/scancode-toolkit'
+    response = github_tags.get_tags(url=url)
+    assert (response[-1] == 'v1.4.2') 
```

Comment:
You should change this to `assert 'v1.4.2' in response`. This is easier to read. 

Additionally, we should test for two additional items:
1) Another version number that is a valid tag in `response`
2) A version number that IS NOT a valid tag in `response`

Reply:
Done @majurg :)